src/main/scala/org/broadinstitute/dsde/firecloud/webservice/UserApiService.scala

```diff
@@ -188,9 +188,10 @@ trait UserApiService extends HttpService with PerRequestCreator with FireCloudRe
         pathEnd {
           get {
             requireUserInfo() { userInfo =>
-              mapRequest(addFireCloudCredentials) {
-                passthrough(UserApiService.remoteGetAllURL.format(userInfo.getUniqueId), HttpMethods.GET)
```

Comment:
As a sanity check, this was originally a Thurloe passthrough but now the response is constructed by Orchestration. Are these identically structured JSON responses? I'm only asking to ensure that backwards-compatibility is maintained.

Reply:
yes, these are identical - the construction we're using is what was being passed through, if that makes sense. we didn't change anything about how the JSON is constructed.